critiquebrainz/frontend/templates/user/info.html

```diff
@@ -5,7 +5,7 @@
 {% block profile_content %}
   {% if user.musicbrainz_id %}
     <div>
-      <img src="{{ url_for('static', filename='favicons/musicbrainz-16.ico') }}" />
+      <img src="{{ url_for('static', filename='favicons/musicbrainz-16.png') }}" />
```

Comment:
If this exact same file(name) is used in all these different places, maybe it would make sense to make it a variable defined somewhere higher up? I'm also thinking ahead to once (in the future) when we move to use [StaticBrainz](http://tickets.musicbrainz.org/browse/MBH-413) and will want to point to that instead - it will be easier to update it in one place, and be less likely to miss somewhere.